src/message.ts

```diff
@@ -1111,17 +1112,55 @@ export class MediaMessage extends Message {
       if (!this.obj.url) {
         throw new Error('no url')
       }
-      log.verbose('MediaMessage', 'stream() url: %s', this.obj.url)
+      log.verbose('MediaMessage', 'readyStream() url: %s', this.obj.url)
       return UtilLib.urlStream(this.obj.url, cookies)
     } catch (e) {
-      log.warn('MediaMessage', 'stream() exception: %s', e.stack)
+      log.warn('MediaMessage', 'readyStream() exception: %s', e.stack)
       Raven.captureException(e)
       throw e
     }
   }
 
-  public forward(room: Room): Promise<boolean>
-  public forward(contact: Contact): Promise<boolean>
+  /**
+   * save file
+   *
+   * @param filePath save file
+   */
+  public async saveFile(filePath: string): Promise<boolean> {
```

Comment:
About the return value `boolean`:

I'd like to suggest to use `void` as the return value. If so, we might want to `throw` an Error if the operation failed.

Reply:
It has been modified.